web_tool_script_3.R

```diff
@@ -169,14 +168,61 @@ if (file.exists(file.path(results_path, portfolio_name, "Bonds_results_map.rda")
     "scenario_geography" = NA_character_
   )
 }
-# bonds_results_map <-read_rds(file.path(results_path, portfolio_name, paste0(portfolio_name, "_Bonds_results_map.rda")))
 
-# bonds_results_map <- equity_results_map %>% filter(portfolio_name == "")
+# load equity stress test data
+if (file.exists(file.path(results_path, portfolio_name, "Equity_results_stress_test.rda"))) {
+  equity_results_stress_test <- read_rds(file.path(results_path, portfolio_name, "Equity_results_stress_test.rda"))
+} else {
+  equity_results_stress_test <- tibble(
+    "investor_name" = NA_character_, "portfolio_name" = NA_character_,
+    "ald_sector" = NA_character_, "technology" = NA_character_,
+    "scenario_geography" = NA_character_, "VaR_technology" = NA_real_,
+    "asset_portfolio_value" = NA_real_, "VaR_Sector" = NA_real_,
+    "scenario_name" = NA_character_, "technology_exposure" = NA_real_,
+    "ector_exposure" = NA_real_, "sector_loss" = NA_real_,
+    "climate_relevant_var" = NA_real_, "portfolio_aum" = NA_real_,
+    "portfolio_loss_percentage" = NA_real_, "year_of_shock" = NA_integer_,
+    "duration_of_shock" = NA_integer_, "production_shock_percentage" = NA_real_
+  )
```

Comment:
Maybe extract as `empty_equity_results_stress_test()` so you can reuse it and it's easier to read.

Reply:
I will do this in another PR as I am checking for NAs in create_interactive_report() atm and that is already merged

Reply:
https://github.com/2DegreesInvesting/PACTA_analysis/issues/212